src/transformStateWithClones.js

```diff
@@ -7,7 +7,43 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const res = [];
+  const status = { ...state };
+  const ActionsTipes = {
+    AddProperties: 'addProperties',
+    RemoveProperties: 'removeProperties',
+    Clear: 'clear',
+  };
+
+  for (const value of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
always better name it in this way :arrow_up: 
for example `for (const person of people)` and so on